packages/plugins/FileService/src/SNSAdaptor/index.tsx

```diff
@@ -52,7 +52,18 @@ const definition: Plugin.SNSAdaptor.Definition = {
                     />
                 )
             },
-            defaultSortingPriority: 2,
+            appBoardSortingDefaultPriority: 2,
+            marketListSortingPriority: 2,
+            icon: <FileServiceIcon />,
+            category: 'dapp',
+            description: {
+                i18nKey: 'plugin_description',
```

Comment:
All i18n keys used in this way (in `I18NField`) need to start with `__`

Reply:
done